cropper/src/main/java/com/canhub/cropper/CropOverlayView.kt

```diff
@@ -92,6 +104,8 @@ class CropOverlayView
     /** The Paint used to darken the surrounding areas outside the crop area.  */
     private var mBackgroundPaint: Paint? = null
 
+    private var mTextLabelPaint: Paint? = null
```

Comment:
this `m` is the old java way. And we didn't remove every where, please use `textLabelPaint`